students/JonSerpas/lesson02/assignment/src/charges_calc.py

```diff
@@ -0,0 +1,140 @@
+'''
+Returns total price paid for individual rentals 
+'''
+import argparse
+import json
+import datetime
+import math
+import logging
+
+# defining the format for logs as they are written to the logfile
+# log_format = '%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s'
+# formatter = logging.Formatter(log_format)
+
+# log_file = datetime.datetime.now().strftime('%Y-%m-%d')+".log"
+# file_handler = logging.FileHandler(log_file)
+
+
+# file_handler.setFormatter(formatter)
+
+# logger = logging.getLogger()
+
+# logger.addHandler(file_handler)
+# # sets the format style for log file output
+# # log_file = datetime.datetime.now().strftime(“%Y-%m-%d”)+’.log’
+
+
+def parse_cmd_arguments():
+    parser = argparse.ArgumentParser(description='Process some integers.')
+    parser.add_argument('-i', '--input',
+                        help='input JSON file',
+                        required=True,)
+    parser.add_argument('-o', '--output',
+                        help='ouput JSON file',
+                        required=True)
+    parser.add_argument('-d', '--debug',
+                        help='set logging level',
+                        required=False)
```

Comment:
Since your argument here is not required, it's generally a good idea for it to have a default. Otherwise anywhere you refer to args.debug, you will get None.